packages/store-ui/src/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import type { InputHTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  isError?: boolean
+}
+
+const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
+  { isError, ...props },
+  ref
+) {
+  const commonProps = {
+    ref,
+    'data-store-input': '',
+    ...props,
+  }
+
+  const errorProp = isError
+    ? {
+        'data-error': '',
+      }
+    : {}
+
+  const allProps = {
+    ...commonProps,
+    ...errorProp,
+  }
```

Comment:
This seems unnecessary, you could just spread both on the input 🤔. What do you think?

Reply:
Yes, I think whatever on that context